topics/admin/tutorials/galaxy-docker/slides.html

```diff
@@ -164,7 +164,7 @@
 Image:
 - recipe that tell how to build your container
 - A filesystem and parameters to use at runtime
-- snapshot of the content of a container: it doesn’t have state and never changes once built
+- snapshot of the content of a container: it doesn'��t have state and never changes once built
```

Comment:
@erasche do you hide some strange characters to be sure we review the PRs? 😄

Reply:
So this accidentally happened because I hit the wrong button during correction. This only happened because of the "pretty" quotes.

Does training material want to use those fancy quotes? or should it use  `'` instead?